integration/manifests/cleaner/cleaner-for-calico.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: danm-cleaner
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: caas:danm-cleaner
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - list
+  - watch
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - endpoints
+  verbs:
+  - create
+  - update
+  - get
+- apiGroups:
+  - danm.k8s.io
+  resources:
+  - danmnets
+  - danmeps
+  - tenantnetworks
+  - clusternetworks
+  verbs: [ "*" ]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: caas:danm-cleaner
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: caas:danm-cleaner
+subjects:
+- kind: ServiceAccount
+  name: danm-cleaner
+  namespace: kube-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: danm-cleaner
+  namespace: kube-system
+  labels:
+    danm: cleaner
+spec:
+  selector:
+    matchLabels:
+     danm: cleaner
+  template:
+    metadata:
+      name: danm-cleaner
+      labels:
+        danm: cleaner
+    spec:
+      serviceAccountName: danm-cleaner
+      containers:
+      - name: danm-cleaner
+        image: cleaner
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - name: calico-config
+          mountPath: /etc/calico/calicoctl.cfg
+          readOnly: true
+        - name: calicoctl
+          mountPath: /usr/local/sbin/calicoctl
+          readOnly: true
+        - name: etcd-certs
+          mountPath: /etc/etcd/ssl
+          readOnly: true
+    volumes:
+    - name: calico-config
```

Comment:
configure only the needed values, from configMap instead?

Reply:
Calicoctl configuration contains only the needed values, which contains database connection type, database endpoints list and client certificate files location to connect with. Thats all. I updated it to using ConfigMap.